pkgs/games/zdoom/zdbsp.nix

```diff
@@ -21,7 +21,7 @@ stdenv.mkDerivation rec {
     description = "ZDoom's internal node builder for DOOM maps";
     homepage = "https://zdoom.org/wiki/ZDBSP";
     license = licenses.gpl2Plus;
-    maintainers = with maintainers; [ertes];
-    platforms = platforms.linux;
+    maintainers = with maintainers; [ ertes ];
```

Comment:
oh man, we need to remove ertes from maintainers someday

Reply:
He was pretty active the last year.

Reply:
https://byorgey.wordpress.com/2018/05/21/ertugrul-soylemez-1985-2018/

Reply:
Sorry to hear that. I will take over maintenance of this package then.

Reply:
@Lassulus would you like to be added to `zdbsp` maintainers?

Reply:
sure, why not